### ISSUE TITLE
Cannot configure multiple "Import" modules with Python plugin

### ISSUE THREAD

**Issue by simonpasquier** [pdxcat/puppet-module-collectd#227]
I have a need to instantiate several Python modules and IIUC this isn't supported by the current Puppet manifest.

Ideally it should be possible to generate this configuration:
````
 <Plugin python>
    ModulePath "/path/to/your/python/modules"
    Import "foo"
    Import "bar"

    <Module foo>
      some_param 1
    </Module>
    <Module bar>
      other_param 1
    </Module>
  </Plugin>
````

**Comment by txaj** [comment]
Feel free to submit a PR. The design pattern to use is similar to some other plugins, like postgresql filecount or curl

**Comment by simonpasquier** [comment]
Yep, I'll try to find some time to do it.

**Comment by blkperl** [comment]
Fixed in v3.4 I think
